code/modules/swarmers/swarmer_act.dm

```diff
@@ -5,6 +5,31 @@
   * Arguments:
   * * S - A reference to the swarmer doing the interaction
   */
+#define DANGEROUS_DELTA_P 250	//Value in kPa where swarmers arent allowed to break a wall or window with this difference in pressure.
+
+/atom/proc/return_wall_delta_p(var/turf/target_turf)	//Finds the greatest difference in pressure across a closed turf.
```

Comment:
Why is this an `/atom` proc instead of a `/turf` one, using `target_turf` as the source?

Reply:
Ah! Just mimicking the other guy. I'll change it!